docs/src/main/asciidoc/security-oidc-code-flow-authentication.adoc

```diff
@@ -266,21 +267,24 @@ The following example shows how you can configure `quarkus-oidc` to support `mTL
 
 [source,properties]
 ----
-quarkus.oidc.tls.verification=certificate-validation
+quarkus.oidc-client.tls.tls-configuration-name=oidc-client
+
+# configure hostname verification if necessary
+#quarkus.tls.oidc-client.hostname-verification-algorithm=NONE
 
 # Keystore configuration
-quarkus.oidc.tls.key-store-file=client-keystore.jks
-quarkus.oidc.tls.key-store-password=${key-store-password}
+quarkus.tls.oidc-client.key-store.jks.path=client-keystore.jks
```

Comment:
not necessarily something to do in this PR, but can we move away from JKS, pretty please???

Seriously, this format is deprecated for a long time. Use p12 instead.

Reply:
I agree; the reason why I used JKS is that I wanted it one to one with the old docs. I'll update it when I push changes that @sberyozkin will suggest later (dropping deprecation or some convetion). Thank you

Reply:
@cescoffier @michalvavrik I don't really mind, I don't recall that the OIDC code insists somewhere on the JKS format

Reply:
May be it was a copy and paste, and there is indeed some code in OidcCommonUtils that does it, not for this PR though

Reply:
@michalvavrik I think you can just update that example to show what @cescoffier strongly recommends

Reply:
FYI @sberyozkin I was referring to this "JKS" code. if this can run also PKCS12, I didn't try it so sorry for confusion. I think it's just JKS.

https://github.com/quarkusio/quarkus/blob/a486b5db0f1feec3088a8addb8002e91955e62ae/extensions/oidc-common/runtime/src/main/java/io/quarkus/oidc/common/runtime/OidcCommonUtils.java#L387

Reply:
Thanks @michalvavrik, yeah, I'll open a bug request to get rid of hard-coding to JKS